KMLConfig.cmake.in

```diff
@@ -4,4 +4,6 @@ set(KML_INCLUDE_DIRS
   "${Boost_INCLUDE_DIR}")
 
 set(KML_LIBRARY_DIR  "${libkml_BINARY_DIR}")
```

Comment:
Since we are now exporting targets, I think this is superfluous? If yes, `KMLConfig.cmake.in` and `KMLConfig-install.cmake.in` can be collapsed into one file, which would be nice :smile:.